### PR TITLE
Covariances

### PR DESCRIPTION
**Pull request by yf297**
special functions are called from boost which is installed through the BH package

### CODE REVIEW

**Comment by yf297** [comment]
need to fix something

**Comment by yf297** [comment]
special functions are called from boost which is installed through the BH package

**Review by joeguinness — CHANGES_REQUESTED** [review]
remove .Rhistory and add space before BH in linkingto

File: .Rhistory [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rcpp::sourceCpp('Desktop/omptest.cpp')
```
Comment (joeguinness) [inline .Rhistory]: delete this file

File: DESCRIPTION [hunk 2]
```diff
@@ -24,14 +24,15 @@ Depends: R (>= 2.10)
...
+    BH
 Suggests:
     fields,
     knitr,
     rmarkdown,
     testthat,
     maps,
     maptools
-LinkingTo: Rcpp, RcppArmadillo
+LinkingTo: Rcpp, RcppArmadillo,BH
```
Comment (joeguinness) [inline DESCRIPTION]: add space before BH

**Review by joeguinness — CHANGES_REQUESTED** [review]

**Review by joeguinness — APPROVED** [review]

**Pull request merged**